data/json/recipes/recipe_others.json

```
@@ -3380,5 +3380,23 @@
     "qualities": [ { "id": "CUT", "level": 2 } ],
     "components": [ [ [ "stick", 1 ] ], [ [ "sheet_canvas", 1 ], [ "sheet_canvas_patchwork", 1 ] ] ],
     "byproducts": [ [ "splinter", 8 ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "result": "hobo_candle",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "survival",
+    "difficulty": 3,
+    "time": "10 m",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 2 } ],
+    "components": [
+      [ [ "gum", 3 ], [ "super_glue", 10 ], [ "bone_glue", 1 ] ],
```

It's 4-to-1 ratio as it's said in https://github.com/CleverRaven/Cataclysm-DDA/blob/cbe022d7971b249af0119a86ef5b66522ebc7d8c/data/json/requirements/materials.json#L9-L12
```suggestion
      [ [ "gum", 3 ], [ "super_glue", 4 ], [ "bone_glue", 1 ] ],
```

---

Hmm, I got that ratio from the axe ring holster. Didn't realize there's a standardized ratio, so this is probably correct.